engine/src/main/java/org/pentaho/di/www/StatusServletUtils.java

```diff
@@ -0,0 +1,78 @@
+/*! ******************************************************************************
+ *
+ * Pentaho Data Integration
+ *
+ * Copyright (C) 2002-2018 by Hitachi Vantara : http://www.pentaho.com
```

Comment:
@pentaho-mracine 
No need to have 2002-2018 on new files...just 2018

Reply:
@ddiroma thanks for noticing that, just a leftover from copying it from another file. Updating now...